packages/sdk-browser-keysigner/package.json

```diff
@@ -0,0 +1,4 @@
+{
+  "name": "@dfns/sdk-browser-keysigner",
```

Comment:
does it need to be a separate package or can this just go into the keysigner package? given it has 0 external lib dependency?

Reply:
done

Reply:
mh though the keysigner package is meant for backend side, no ? It uses nodejs `crypto` modue

Reply:
If you don't use the AsymetricSigner then it doesn't matter

Reply:
yeah as long as it's not bringing in unnecessary dependencies I think it's ok, since both are just key signers. ideally if we can have a single key signer that works in both environments, that would be even better, but probably not worth the effort.

Reply:
In theory node crypto is faster than browser subtle crypto, keep 2 for now 😅

Reply:
Ok I guess what's weird is that someone can install that module and use one thing, but not another or it'll fail... a bit weir I think